docs/supported/index.rst

```diff
@@ -0,0 +1,267 @@
+========================
+Supported Configurations
+========================
+
+Supported Cluster Managers
+==========================
+
+.. raw:: html
+
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
+			<img src="/_images/docker2x.png" alt="Docker logo"/>
+			<span>Docker, Swarm, Compose <em>Fastest</em></span>
+	        <a href="docker-integration/" class="button button--fast">Learn more</a>
+	    </div>
+	    
+	    <div class="pod-boxout pod-boxout--orchestration">
+			<img src="/_images/kubernetes2x.png" alt="Kubernetes logo"/>
+			<span>Kubernetes</span>
+	        <a href="kubernetes-integration/" class="button">Learn more</a>
+	    </div>
+	    
+	    <div class="pod-boxout pod-boxout--orchestration">
+			<img src="/_images/mesos2x.png" alt="mesos logo"/>
+			<span>Mesos</span>
+	        <a href="mesos-integration/" class="button">Learn more</a>
+	    </div>
+	</div>
+	
+	 <div class="pod-boxout pod-boxout--minor pod-boxout--orchestration">
+		<span><img src="/_images/icon-question2x.png" aria-hidden="true" alt=""/>&nbsp; You can install Flocker without a specific cluster manager, click here to learn more:</span>
+        <a href="/flocker-standalone/" class="button">Learn more</a>
+    </div>
+
+.. _storage-backends:
+
+Supported Infrastructure & Storage
+==================================
+
+The sections below list the supported infrastructure and storage options.
+
+**Community Developed** drivers are those provided and supported by our community partners.
```

Comment:
Can we please consider removing references to storage drivers? Drivers are implementation detail, and the user reading this page would be primarily interested in knowing whether or not his storage backend is a supported configuration. Rewording ``Community Developed`` to ``Community Supported`` might be more useful well. Thanks.